cmd/root.go

```diff
@@ -55,6 +56,36 @@ func RunCommand() *cobra.Command {
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
 			output.SetOut(cmd.OutOrStdout())
 			output.SetErr(cmd.ErrOrStderr())
+
+			// Load config from the file (or default if it doesn't exist)
+			configHome, err := getConfigHome(opts)
+			if err != nil {
+				output.Errorf("Failed to read base config directory")
+				output.Infof("Use the --config flag or set the $%s environment variable to provide a default", constants.KitopsHomeEnvVar)
+				output.Debugf("Error: %s", err)
+				return errors.New("exit")
+			}
+
+			configPath := filepath.Join(configHome, "config.json")
```

Comment:
Please move the filename here (`config.json`) to the [constants package](https://github.com/jozu-ai/kitops/blob/4a7308af50bf3342a1c10d610889e1a78a156b19/pkg/lib/constants/consts.go#L36-L41)

Reply:
Refactored